src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+@import 'main';
+@import 'reset';
+@import 'header';
+@import 'variables';
+@import 'stars';
+@import 'card';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
-  margin: 0;
+  font-family: $font-family;
+  background-color: $body-bg-color;
+  height: 100vh;
```

Comment:
Consider moving these styles to a separate file to keep the index clean and clear